docs/docs/how-tos/nebari-upgrade.md

```diff
@@ -118,45 +191,29 @@ project_name: mynebarinew
 domain: nebari.myproj.com
 ```
 
-It is also a good time to upgrade your version of Kubernetes. Look for the `kubernetes_version` field within the cloud
-provider section of the `nebari-config.yaml` file and increase it to the latest.
+It is also a good time to upgrade your version of Kubernetes. Look for the `kubernetes_version` field within the cloud provider section of the `nebari-config.yaml` file and increase it to the latest.
 :::
 
 ### Redeploy Nebari
 
-If you are deploying Nebari from your local machine (not using CI/CD) then you will now have a `nebari-config.yaml` file
-that you can deploy.
+For local deployments, run the following
 
 ```shell
-nebari deploy -c nebari-config.yaml
+nebari deploy -c nebari-config.yaml   
 ```
 
-At this point you may see an error message saying that deployment is prevented due to the `prevent_deploy` setting in
-your YAML file. This is a safeguard to ensure that you only proceed if you are aware of possible breaking changes in the
-current upgrade.
-
-For example, we may be aware that you will lose data due to this upgrade, so need to note a specific upgrade process to
-keep your data safe. Always check the release notes of the release in this case and get in touch with us if you need
-assistance. For example, you may find that your existing cluster is intentionally deleted so that a new replacement can
-be deployed instead, in which case your data must be backed up so it can be restored after the upgrade.
-
-**When you are ready, remove the `prevent_deploy` setting from the config file, and run the `nebari deploy -c nebari-config.yaml` again.**
+If there is a `prevent_deploy` setting in your YAML file, you will remove this after verifying all is correct in your config file.
 
 ### CI/CD: render and commit to git
 
-For CI/CD (GitHub/GitLab) workflows, then as well as generating the updated `nebari-config.yaml` files as above, you will
-also need to regenerate the workflow files based on the latest `nebari` version's templates.
-
-With the newly upgraded `nebari-config.yaml` file, run:
+For CI/CD (GitHub/GitLab) workflows, update the workflow files.  
+Run the following with the updated `nebari-config.yaml` file:
 
 ```shell
 nebari render -c nebari-config.yaml
 ```
 
-(Note that `nebari deploy` would perform this render step too, but will also immediately redeploy your Nebari.)
-
-Commit all the files (`nebari-config.yaml` and GitHub/GitLab workflow files) back to the remote repo. All files need to be
-committed together in the same commit.
+Commit all the files (`nebari-config.yaml` and GitHub/GitLab workflow files) back to the remote repo. All files need to be committed together in the same commit.
 
 :::warning Breaking upgrades
 For upgrades from older versions to v0.4, you will need to do the following steps. Everyone else is done!
```

Comment:
@ppwadhwa Since we are inside the sub-heading ["Upgrade Nebari from pre-0.4.0 to 0.4.0"](https://deploy-preview-288--nebari-docs.netlify.app/docs/how-tos/nebari-upgrade/#upgrade-nebari-from-pre-040-to-040), I believe these steps are mandatory for everyone following the instructions here?

I'm wondering if we can remove this warning or move the warning to a separate block before the steps, and keep all steps in the main document flow? -- Primarily because if we have heading-tags within the warning element, they don't get displayed in the right sidebar table of contents (screenshot attached).

![Screenshot 2023-03-13 at 6 52 23 PM](https://user-images.githubusercontent.com/33131404/224714277-a60e8501-cd69-44e6-b38f-d78aa6f1ec3f.png)